### PR TITLE
CONTRIBUTING: format Markdown and CI runtime information

### PR DESCRIPTION
**Pull request by aparcar**
Also surround binaries and variable names with ticks so it's clearer
what they are.

Markdown formating via `remark-cli`.

This looks like a big change but if you check the diff of rendered Markdown, it's trivial.

### CODE REVIEW

**Comment by aparcar** [comment]
@jefferyto please have a look

**Comment by jefferyto** [comment]
The diff is a bit hard to read (just has a quick glance on mobile)... perhaps you can split the remark formatting changes into one commit and other additions/changes into other commit(s)?

**Comment by aparcar** [comment]
It's a bit tedious to fiddle both into to commits again, do you mind looking at the rendered version?

image

**Comment by jefferyto** [comment]
```bash
git checkout contrib
cp CONTRIBUTING.md CONTRIBUTING.md.new
git checkout master
cp CONTRIBUTING.md CONTRIBUTING.md.old
git checkout contrib
mv CONTRIBUTING.md.old CONTRIBUTING.md
<run remark-cli>
git add CONTRIBUTING.md
git commit --amend
mv CONTRIBUTING.md.new CONTRIBUTING.md
git add CONTRIBUTING.md
git commit --signoff
```

The commit will be viewed by people far into the future, not necessarily through GitHub's web interface. If this was a personal repo I wouldn't mind how you do your changes. This is a public repo though, let's spend the extra 2 minutes to do it right.

IMHO formatting changes (i.e. no functional change) should always be done in separate commits. They are hard enough to read as it is, adding additional changes makes it unnecessarily difficult to know what was changed.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,130 +1,206 @@
...
+
+## Continuous Integration
+
+To simplify review and require less human resources, a CI tests all packages.
+Passing CI tests are not a hard requirement but a good indicator what the
+Buildbots will think about the proposed patch.
+
+The CI builds modified packages for multiple architectures using the latest
+snapshot SDK. For supported architectures (`aarch64_generic`,
+`arm_cortex`-a15_neon-vfpv4 ,`i386_pentium4` and `x86_64`) an additional
```
Comment (jefferyto) [inline CONTRIBUTING.md:193]:
```suggestion
`arm_cortex-a15_neon-vfpv4`, `i386_pentium4` and `x86_64`) an additional
```

Comment (aparcar) [reply CONTRIBUTING.md]: Thanks.

**Comment by aparcar** [comment]
This JS npm [foo](https://github.com/remarkjs/remark/blob/HEAD/doc/plugins.md#list-of-plugins) gave me a hard time but I think now it's somewhat okayish.

**Comment by aparcar** [comment]
@jefferyto ping?

**Review by neheb — APPROVED** [review]

**Pull request merged**